### PR TITLE
Invoke-DSCResource 

### LINKED ISSUE #4780: Document Invoke-DscResource
Issue Details
-----------------

https://github.com/PowerShell/PowerShell/pull/10516
https://github.com/PowerShell/PowerShell-RFC/pull/214

Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by theJasonHelmick**
# PR Summary
Fixes #4780 

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+ms.date: 01/10/2020
+online version: https://docs.microsoft.com/powershell/module/psdesiredstateconfiguration/invoke-dscresource?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: Invoke-DscResource
+---
+
+# Invoke-DscResource
+
+## SYNOPSIS
+
```
Comment (sdwheeler) [inline reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: Remove this blank line

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Runs a method of a specified PowerShell Desired State Configuration (DSC) resource.
+
+## SYNTAX
+
+```powershell
```
Comment (sdwheeler) [inline reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: Remove language tag for syntax blocks.

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+## EXAMPLES
+
+### Example 1: Invoke the Set method of a resource by specifying its mandatory properties
+
+This command invokes the **Set** method of a resource named Log and specifies a **Message** property
+for it.
+
+```powershell
+PS C:\> Invoke-DscResource -Name Log -Method Set -Property @{Message = 'Hello World'} -ModuleName PSDesiredStateConfiguration
```
Comment (sdwheeler) [inline reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: Don't use prompts in examples unless needed for context

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md [hunk 4]
```diff
@@ -0,0 +1,448 @@
+---
+title: What's New in PowerShell 7.0
```
Comment (sdwheeler) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md]: remove this file from your PR

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+```
+
+### Example 2: Invoke the Test method of a resource for a specified module
+
+This command invokes the **Test** method of a resource named WindowsProcess, which is in the module
+named PSDesiredStateConfiguration.
```
Comment (chasewilson) [inline reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: Module names are bold

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
+-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
+-WarningAction, and -WarningVariable. For more information, see
+[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (chasewilson) [inline reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: this should be:

This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, 
`-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, 
`-WarningAction`, and `-WarningVariable`. For more information, see 
[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md (hunk 5 above)
Comment (chasewilson) [reply reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: Adjusting by suggestion

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+```powershell
+Invoke-DscResource -Name Log -Method Set -ModuleName PSDesiredStateConfiguration -Property @{
+  Message = 'Hello World'
+}
+```
+
+### Example 2: Invoke the Test method of a resource for a specified module
+
+This command invokes the **Test** method of a resource named WindowsProcess, which is in the module
+named PSDesiredStateConfiguration.
```
Comment (chasewilson) [inline reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]:
```suggestion
named **PSDesiredStateConfiguration**.
```

File: reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md (hunk 6 above)
Comment (sdwheeler) [reply reference/7.0/PSDesiredStateConfiguration/Invoke-DscResource.md]: This change is not needed.

**Pull request merged**